docs/getting-started.md

```diff
@@ -81,7 +81,7 @@ option is the only required option in your configuration file.
 Now add a second page to your documentation:
 
 ```bash
-curl 'https://jaspervdj.be/lorem-markdownum/markdown.txt' > docs/about.md
+curl 'https://jaspervdj.be/lorem-markdownum/markdown.txt?no-external-links=on' > docs/about.md
```

Comment:
@tomchristie If this looks better:

```suggestion
curl -L 'https://tinyurl.com/lorem-markdownum' > docs/about.md
```

? (I kinda dislike relying on some redirector changing its behavior in the future breaking this etc.)